src/utils/GetStyledCalendarElement/GetStyledCalendarElement.test.js

```diff
@@ -45,6 +37,15 @@ describe("GetStyledCalendarElement", () => {
 
         expect(calendarHeader.childNodes[0].innerText).to.be.an("undefined");
       });
+
+      it("header p returns a 15px height", () => {
```

Comment:
Please read our universal testing-guide if you haven't done it yet:
- https://github.com/c-hive/guides/blob/master/universal/testing.md

Either way, this description may fit better in this case.

```suggestion
      it("sets the height of <p /> to 15px", () => {
```